.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
    release:
      executor: ubuntu-build
      environment:
-       BAZEL_TEST_TARGETS: "//test/extensions/access_loggers/wasm/... //test/extensions/filters/http/wasm/... //test/extensions/wasm/..."
+       BAZEL_TEST_TARGETS: "//test/extensions/access_loggers/wasm/... //test/extensions/filters/http/wasm/... //test/extensions/wasm/... //test/extensions/commmon/wasm/..."
```

Comment:
Type, `s/commmon/common/` (here and everywhere else).

Reply:
done